int32 DIGIN_2 
float64 AIN0_2
float64 AIN1_2
